pages/Messages/Signatures.md

```diff
@@ -27,16 +27,19 @@ Optionally, anonymous messages may be provided with a zero hash in place of the
 
 The process for generating signatures for DSNP announcements consists of three distinct steps:
 
-1. Serialize the DSNP announcement
+1. Serialize the DSNP announcement, if not encrypted
 1. Hash the serialized string
 1. Sign the hash
 
-Before serializing, keys in the DSNP object should be sorted alphabetically using [default JavaScript string comparison](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator) order.
-Serialization should be done as described in the [Serialization spec](/Identity/Serialization).
-Next, the serialized string should be hashed using [keccak-256](https://en.wikipedia.org/wiki/SHA-3).
-Finally, the fixed length hash generated can then be signed using [secp256k1](https://google.com/search?hl=en&q=secp256k1) and the publishing user's private key.
+For encrypted messages, the encrypted bytes of the `dsnpData` field can be used directly without any serialization.
 
-Once the signature is generated, it should be provided along with the message contents to the announcer for inclusion in the next available batch.
+For non-encrypted messages, keys in the `dsnpData` object should be sorted alphabetically using [default JavaScript string comparison](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator) order to guarantee a single, unique string for any given object.
```

Comment:
This references `Intl.Collator` which permits customization by language. The MDN page suggests this defaults to the default locale, which will cause inconsistent serializations. Perhaps this should specify the language as well (which I think would have to be `en_US`).